command/command.go

```diff
@@ -613,8 +612,7 @@ func (app *App) UpdateHostSpecs() {
 		return
 	}
 
-	err = app.API.UpdateHost(app.Host.ID, mkr.UpdateHostParam(hostParam))
-
+	_, err = app.API.UpdateHost(app.Host.ID, (*mkr.UpdateHostParam)(hostParam))
```

Comment:
This cast is required?

Reply:
Yes, this is required. hostParam is typed as `*mkr.CreateHostParam`.